packages/flutter_tools/templates/create/.gitignore.tmpl

```diff
@@ -1,11 +1,17 @@
 .DS_Store
-.atom/
 .dart_tool/
+
+.atom/
 .idea
 .vscode/
+
 .packages
 .pub/
+packages
```

Comment:
We should remove `packages` now actually. It's for the old package system.

Reply:
Done.